velero-v1.12.0-linux-amd64/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
Did you mean to add this dir?

Reply:
this was done automatically when running the updated kubevirtci..

Reply:
Yeah at some point velero is downloaded but I don't think we want to add it to the repo.  Maybe put something in gitignore?

Reply:
@mhenriks add the gitignore and deleted an existing velero repo we had in here.